packer/epoch.json

```diff
@@ -0,0 +1,74 @@
+{
+  "variables": {
+    "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
+    "aws_secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}"
+  },
+  "builders":
+  [
+    {
+      "type": "amazon-ebs",
+      "region": "us-west-2",
+      "source_ami_filter": {
+        "filters": {
+          "virtualization-type": "hvm",
+          "name": "ubuntu/images/*ubuntu-xenial-16.04-amd64-server-*",
+          "root-device-type": "ebs"
+        },
+        "owners": ["099720109477"],
+        "most_recent": true
+      },
+      "instance_type": "t2.micro",
+      "ssh_username": "ubuntu",
+      "ami_name": "epoch-ubuntu-16.04-v{{timestamp}}",
+      "ami_regions": ["eu-central-1", "ap-southeast-1"],
+      "force_deregister": true,
+      "force_delete_snapshot": true
+    },
+    {
+      "type": "openstack",
+      "ssh_username": "master",
+      "image_name": "epoch-ubuntu-16.04-v{{timestamp}}",
+      "source_image": "5566886d-0d9d-4d86-94ff-55597169e15a",
```

Comment:
The build in provider Ubuntu 16.04 image, shall we use 16.04.3 (also build by the OS provider) to speed the build up?

Reply:
The OS provider image 16.04.3 is too old, I've used our own private 16.04.4 image in 80603c8d024281a626c19ab0081f6007efd7d418

Reply:
New image with fixed cloud-init: c923ab5df6ac795aec6850e17140a9740a638f65